<launch>
  
  <node pkg="render_robot_urdf" type="render_robot_urdf" name="render_robot_urdf" output="screen" clear_params="true">

      <param name="with_color" type="bool" value="true" />
      <param name="with_normals" type="bool" value="true" />

      <param name="model_param_name" type="string" value="robot_description" />

      <!-- set this to your robot description -->
      <param
        name="robot_description"
        command="$(find xacro)/xacro '$(find render_robot_urdf)/data/urdf/comau_nolimits_nobase.urdf'"/>
  </node>

  <node pkg="render_robot_urdf" type="render_robot_urdf_test" name="render_robot_urdf_test" output="screen" clear_params="true">

  </node>

</launch>
